SlackAPI.Tests/UserUIInteraction.cs

```diff
@@ -40,25 +40,26 @@ public void TestGetAccessToken()
                 driver.FindElement(By.Id("password")).SendKeys(authPassword);
                 driver.FindElement(By.Id("signin_btn")).Click();
 
-                var uri = SlackClient.GetAuthorizeUri(clientId, SlackScope.Identify);
+                var slackClientHelpers = new SlackClientHelpers();
```

Comment:
It shows exactly the API break here. Introducing SlackClientHelpers was required to be able to call these methods with proxy support